java/ql/src/semmle/code/java/security/JWT.qll

```diff
@@ -0,0 +1,119 @@
+/** Provides classes for working with JWT libraries. */
```

Comment:
```suggestion
/** Provides classes for working with JSON Web Token (JWT) libraries. */
```

Reply:
Applied in a046d75. While I was at it, I made the same change in other comments too.